db/internal_stats.cc

```diff
@@ -45,6 +45,7 @@ const std::map<LevelStatType, LevelStat> InternalStats::compaction_level_stats =
         {LevelStatType::READ_MBPS, LevelStat{"ReadMBps", "Rd(MB/s)"}},
         {LevelStatType::WRITE_MBPS, LevelStat{"WriteMBps", "Wr(MB/s)"}},
         {LevelStatType::COMP_SEC, LevelStat{"CompSec", "Comp(sec)"}},
+        {LevelStatType::COMP_CPU_SEC, LevelStat{"CompCPUSec", "CompCPU(sec)"}},
```

Comment:
Maybe call it something like `CompMergeCPU` or something like so that we can explain that it may not include file read and write?